pkgs/os-specific/linux/nvidia-x11/builder.sh

```diff
@@ -33,11 +33,11 @@ installPhase() {
     # since version 391, 32bit libraries are bundled in the 32/ sub-directory
     if [ "$i686bundled" = "1" ]; then
         mkdir -p "$lib32/lib"
-        cp -prd 32/*.so.* 32/tls "$lib32/lib/"
```

Comment:
Is it safe to remove? The same build script is also used for `stable_390`, this change might break it.
It might make sense to have a separate script for 390 or make the copy optional, if the file is there.

Reply:
Forgot that the build script was shared, though it seems like I have to recompile the entire kernel to see if it copies over now.